src/containers/subjects/SubjectFeatureTable/index.tsx

```diff
@@ -42,6 +43,17 @@ function SubjectFeatureTable(props: Props) {
     return <img src={item} alt={''} style={{ width: '100%', display: 'block' }} />;
   };
 
+  const caption = (item: string) => {
+    const filename = new URL(item).pathname.split('/').pop();
+    return (
+      <React.Fragment>
+        <div className='flex flex-wrap align-items-center justify-content-center'>
+          <h4 className='mb-1'>{filename}</h4>
```

Comment:
```suggestion
          <h4 className='mb-1 text-overflow-ellipsis overflow-hidden'>{filename}</h4>
```

Seems that at long caption could exceed the length of the box. This should hide it out.

<img width="194" alt="Screenshot 2023-03-27 at 11 40 29" src="https://user-images.githubusercontent.com/61998484/227815595-ed35b711-3c1e-4f3b-8ec4-58af339ff8cf.png">

Reply:
😢 I know!! Thought, you won't find it! 😭 _(hide & seek)_ Not so perfected there yet.

One for 2.2/Ocicat backlog css tuning; when we shall introduce multi-themes switcher. Let close one eye 🙈 for now, k? 🙏

Reply:
Maybe just commit the suggestion there, then additional css tuning can be added on `Ocicat`?

Reply:
Yah. Let's Ocicat! CSS tuning shall go across multiple themes, yes. 🙂